configs/config.dell_optiplex_9010_sff_uefi

```diff
@@ -1,18 +1,29 @@
 CONFIG_LOCALVERSION="v0.1.0"
-CONFIG_USE_OPTION_TABLE=y
+CONFIG_OPTION_BACKEND_NONE=y
 CONFIG_VENDOR_DELL=y
 CONFIG_FMDFILE="src/mainboard/dell/snb_ivb_workstations/default.fmd"
 CONFIG_ONBOARD_VGA_IS_PRIMARY=y
 CONFIG_BOARD_DELL_OPTIPLEX_9010=y
+CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x100000
+CONFIG_EDK2_BOOT_TIMEOUT=3
 CONFIG_PCIEXP_CLK_PM=y
 CONFIG_EDK2_BOOTSPLASH_FILE="3rdparty/dasharo-blobs/dasharo/bootsplash.bmp"
 CONFIG_TPM_MEASURED_BOOT=y
+CONFIG_DRIVERS_EFI_VARIABLE_STORE=y
+# CONFIG_USE_PC_CMOS_ALTCENTURY is not set
+CONFIG_BOOTMEDIA_LOCK_CONTROLLER=y
+CONFIG_BOOTMEDIA_LOCK_WPRO_VBOOT_RO=y
+CONFIG_BOOTMEDIA_SMM_BWP=y
+CONFIG_DEFAULT_CONSOLE_LOGLEVEL_0=y
```

Comment:
Doesn't this also stop logging to CBMEM?

Reply:
Nope, it doesn't. Look at other configs. The cbmem log is always filled with loglevel 7 or 8 messages.